src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -857,3 +862,12 @@ def transform_column(self, name: str, transformer: Callable[[Row], Any]) -> Time
             ),
             time_name=self.time.name,
         )
+
+    def plot_lag(self, lag: int) -> Image:
```

Comment:
`plot_lagplot` would be more consistent with the plotting methods in `Table` (`plot_<name_of_plot>`).

Reply:
named it plot_lagplot